features/checkout/order_promotion/order_promotion_integrity_validation.feature

```diff
@@ -16,9 +16,19 @@ Feature: Order promotions integrity
 
     @ui
     Scenario: Preventing customer from completing checkout with already expired promotion
-        Given I added product "PHP T-Shirt" to the cart
+        Given this promotion gives "$10.00" discount to every order
+        And this promotion expires tomorrow
+        And I added product "PHP T-Shirt" to the cart
         And I have proceeded selecting "Offline" payment method
         And this promotion has already expired
         When I confirm my order
         Then I should be informed that this promotion is no longer applied
         And I should not see the thank you page
+
+    @ui @promo_validation
```

Comment:
`@promo_validation` not needed as we didn't change anything in behat suites configuration